vdirsyncer/storage/ldap.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+import logging
+import ldap3
+
+from .base import Item, Storage
+from .. import exceptions
+
+ldap_logger = logging.getLogger(__name__)
+
+
+class LDAPStorage(Storage):
+
+    __doc__ = '''
```

Comment:
Also the `__doc__` is only necessary if we need to include variables from other strings.